### PR TITLE
Adicionei uma imaginacao no qual faria uma nova app

### PR DESCRIPTION
**Pull request by pattymargalho**

### CODE REVIEW

**Comment by pattymargalho** [comment]
Please review this, @casey-collab

File: portugues/amigo/incomodar/reinventar/reinventar-app.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Procuro um amigo "geek" para colaborar comigo e fazemos uma versão melhorada sem o bug :)
```
Comment (casey-collab) [inline portugues/amigo/incomodar/reinventar/reinventar-app.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pattymargalho** [comment]
Please review this again, @casey-collab

File: portugues/colaborando.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Você:
 
 [Cria um aplicativo melhor](cria/melhor.md)
 
+[Procura um amigo geek e reinventa um app bem melhor!](reinventar-app.md)
```
Comment (casey-collab) [inline portugues/colaborando.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: portugues/amigo/incomodar/reinventar/reinventar-app.md (hunk 1 above)
Comment (casey-collab) [inline portugues/amigo/incomodar/reinventar/reinventar-app.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by pattymargalho** [comment]
Please review this again, sorry! @casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**